manager/manager.go

```diff
@@ -727,3 +565,188 @@ func (m *Manager) rotateRootCAKEK(ctx context.Context, clusterID string) error {
 	})
 
 }
+
+// startBeingALeader promote itself to be a leader,
+// creating the cluster and the node on the store,
+// initializing default config and starts: Orchestrators, taskReaper, scheduler, allocator and keyManager
+// then it checks if they are working.
```

Comment:
It doesn't really check if they're working. The goroutines log an error if one of the `Run` function exist, but that could happen long after this function has returned.